src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorTool.java

```diff
@@ -57,6 +57,13 @@ public static void main(String[] args) throws Exception {
             .hasArg()
             .argName("OUTPUT-DIRECTORY")
             .build());
+    options.addOption(
```

Comment:
OK, just be aware when you do, you'll potentially need to support this flag as well for a little bit until I change the scripts to run `discoGen` to actually send out the info as yaml files rather than this long file.

With any  luck, I can have that coded earlier, though, and it'll just be a flag flip from the script side.